python/ray/data/dataset.py

```diff
@@ -480,7 +480,7 @@ def map_batches(
             >>> ds = ds.map_batches(map_fn_with_large_output)
             >>> ds
             MapBatches(map_fn_with_large_output)
-            +- Dataset(num_blocks=1, num_rows=1, schema={item: int64})
+            +- Dataset(num_blocks=..., num_rows=1, schema={item: int64})
```

Comment:
For my understanding, what does this change mean?

Reply:
We have a test rule where ellipsis can match any value.